python/identifiers.cpp

```diff
@@ -47,14 +57,14 @@ void register_identifiers(pybind11::module& m) {
         .value("spike_source", arb::cell_kind::spike_source,
             "Proxy cell that generates spikes from a spike sequence provided by the user.");
 
-    pybind11::enum_<arb::backend_kind>(m, "backend",
+    py::enum_<arb::backend_kind>(m, "backend",
         "Enumeration used to indicate which hardware backend to execute a cell group on.")
         .value("gpu", arb::backend_kind::gpu,
             "Use GPU backend.")
         .value("multicore", arb::backend_kind::multicore,
             "Use multicore backend.");
 
-    pybind11::enum_<arb::binning_kind>(m, "binning",
+    py::enum_<arb::binning_kind>(m, "binning",
```

Comment:
I deliberately chose `pybind11` as the namespace. By doing this we are now inconsistent (you are probably going to show me one spot where I used `py` instead of `pybind11` now.

Reply:
Yeah, there were multiple cases of using `py::` instead of `pybind11::`, so I went with the shorter for consistency and brevity.